assets/edudoc-style.css

```diff
@@ -1,61 +1,80 @@
-/* If this is updated, update the styling of the body in edudoc-header.html too */
-body {
-  margin: 0;
-  padding: 0;
-  font-family: arial;
-  font-size: 18px;
+:root {
+  font-size: 11pt;
+}
+
+/* This is used by weasyprint */
+@page {
+  margin: 13mm 15mm;
+  padding: 2mm 0 0;
+  border-top: black solid 1px;
+
+  @top-left {
+    content: element(headerTitle);
+  }
+
+  @top-center {
+    content: element(headerLogo);
+  }
+
+  @top-right {
+    content: element(headerDate);
+  }
+
+  @bottom-center {
+    content: counter(page);
+  }
```

Comment:
Where do the variables like `headerTitle`, `page`, etc. come from?

Reply:
These are just custom names I chose, and they're used below inside of the `@media print` block